zilliqa/src/api/others.rs

```diff
@@ -0,0 +1,36 @@
+// This file contains API calls implementation that naturally didn't fit in other namespaces
+use std::{
+    collections::HashMap,
+    sync::{Arc, Mutex},
+};
+
+use anyhow::Result;
+use jsonrpsee::{types::Params, RpcModule};
+
+use super::types::eth;
+use crate::{api::types::eth::Transaction, node::Node};
+
+pub fn rpc_module(node: Arc<Mutex<Node>>) -> RpcModule<Arc<Mutex<Node>>> {
+    super::declare_module!(node, [("txpool_content", txpool_content)])
+}
+
+fn txpool_content(_params: Params, node: &Arc<Mutex<Node>>) -> Result<Option<eth::TxPoolContent>> {
```

Comment:
Might now be a good time to document this API call? 'cos otherwise someone (read: me) is going to have to reverse-engineer what you intended it to do, figure out whether it does it, and then document it ..

Reply:
Very good point, thanks for the reminder.

Reply:
thanks! Yep - that'll do .. there are a few formatting things I'll fix up when I get there, that you couldn't know about until the PR I'm about to submit! Thank you!